config/manager/manager.yaml

```diff
@@ -18,7 +18,6 @@ spec:
       containers:
       - args:
         - --leader-elect
-        - --logtostderr
```

Comment:
can this be problematic? seems like something that changed on controller-gen maybe and just re-generated? not sure

Reply:
https://cluster-api.sigs.k8s.io/developer/providers/v1.3-to-v1.4.html#other mentions the removal of these deprecated flags from `component-base` which was necessary to get the CAPV manager running.